platform/android/java/lib/res/values-zh-rCN/strings.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="text_paused_cellular">您想启用通过蜂窝连接下载吗？ 根据您的数据计划，这可能会花费您金钱。</string>
+    <string name="text_paused_cellular_2">如果您选择不启用通过蜂窝连接下载，则当 Wi-Fi 可用时，下载将自动恢复。</string>
```

Comment:
For overall consistency with other translated parts of Godot, please use `你` instead of `您` for the second personal pronoun.

Also applies to other strings.